metastats-application/src/main/java/life/qbic/metastats/cli/MetaStatsTool.java

```diff
@@ -26,13 +34,41 @@ public MetaStatsTool(final MetaStatsCommand command) {
     public void execute() {
         // get the parsed command-line arguments
         final MetaStatsCommand command = super.getCommand();
-        /**OpenBisSession session =
-        DatabaseGateway db = new OpenBisSearch(IApplicationServerApi,"")
-        ProjectSpecification spec = new RequestExperimentData()
+        // get properties
+        LOG.debug(command.conf);
+        JsonParser experimentProps = new JsonParser(command.conf);
+        Map credentials = experimentProps.parse();
 
-        MetaStatsController msc = new MetaStatsController(command.conf,command.projectCode);*/
+        JsonValidator validator = new JsonValidator("/model.schema.json");
 
-    }
+        //define output classes
+        MSMetadataPackageOutput metaStatsPresenter = new MetaStatsPresenter(new TSVFileCreator());
+
+        experimentProps = new JsonParser("openbisToMetastatsExperiment.json");
+        Map expMappingInfo = experimentProps.parseStream();
+
+        JsonParser sampleProps = new JsonParser("openbisToMetastatsSample.json");
+        Map sampleMappingInfo = sampleProps.parseStream();
+
+        PropertiesMapper mapper = new OpenBisMapper(expMappingInfo, sampleMappingInfo);
```

Comment:
```suggestion
        PropertiesMapper mapper = new OpenBisMapper(experimentMappingInfo, sampleMappingInfo);
```
exp -> expression
exp -> expected
exp -> experimental

I think the additional 6 letters are worth the information they contain.